### PR TITLE
20 add api error handling

### LINKED ISSUE #20: Add API error handling
E.g. messages shown to the UI if a login or profile update request fails

### PR DESCRIPTION
**Pull request by ctcooper**
closes #20 

# Description #
Create an error message when login fails

# Testing #

1. Got to Login Page
2. Type in invalid email and password
3. Validate error message 'Invalid login credentials' appears

# Screenshots #
Screen Shot 2022-10-27 at 6 10 36 PM

### CODE REVIEW

File: src/lib/Auth.svelte [hunk 1]
```diff
@@ -23,6 +25,8 @@
       if (error) throw error
     } catch (error) {
       console.error(error)
+      console.log('This is the error', error)
```
Comment (ubercj) [inline src/lib/Auth.svelte]: Do we really need to log the error to console twice here?

**Review by ubercj — CHANGES_REQUESTED** [review]
This will make the user experience a lot better, thank you! If you'll get rid of that duplicate log statement, I'll get this merged

File: src/lib/Auth.svelte (hunk 1 above)
Comment (ctcooper) [reply src/lib/Auth.svelte]: Good catch, I thought I had removed that.  We shouldn't ever want any `console.log`'s typically in production.

Comment (ctcooper) [reply src/lib/Auth.svelte]: Pushed up the change to remove it.

**Review by ubercj — APPROVED** [review]
Looks good, thanks 👍 

And good point. I might take a look later at setting a flag so errors are only logged in the dev environment.

**Pull request merged**